[2022-10-26T00:39:23.402+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.insert_dimCategorias manual__2022-10-26T00:31:19.166146+00:00 [queued]>
[2022-10-26T00:39:23.423+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.insert_dimCategorias manual__2022-10-26T00:31:19.166146+00:00 [queued]>
[2022-10-26T00:39:23.424+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-26T00:39:23.425+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 3
[2022-10-26T00:39:23.425+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-26T00:39:23.462+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): insert_dimCategorias> on 2022-10-26 00:31:19.166146+00:00
[2022-10-26T00:39:23.473+0000] {standard_task_runner.py:54} INFO - Started process 3694 to run task
[2022-10-26T00:39:23.479+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'sales_datawarehouse', 'insert_dimCategorias', 'manual__2022-10-26T00:31:19.166146+00:00', '--job-id', '613', '--raw', '--subdir', 'DAGS_FOLDER/sales_dw.py', '--cfg-path', '/tmp/tmp1vyp2zpj']
[2022-10-26T00:39:23.482+0000] {standard_task_runner.py:83} INFO - Job 613: Subtask insert_dimCategorias
[2022-10-26T00:39:23.484+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/sales_dw.py
[2022-10-26T00:39:24.782+0000] {task_command.py:384} INFO - Running <TaskInstance: sales_datawarehouse.insert_dimCategorias manual__2022-10-26T00:31:19.166146+00:00 [running]> on host 83cd0ab98720
[2022-10-26T00:39:24.965+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=pedromenegat
AIRFLOW_CTX_DAG_ID=sales_datawarehouse
AIRFLOW_CTX_TASK_ID=insert_dimCategorias
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T00:31:19.166146+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-26T00:31:19.166146+00:00
[2022-10-26T00:39:25.240+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-10-26T00:39:25.267+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=sales_datawarehouse, task_id=insert_dimCategorias, execution_date=20221026T003119, start_date=20221026T003923, end_date=20221026T003925
[2022-10-26T00:39:25.343+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-26T00:39:25.398+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
